glean-core/src/storage/mod.rs

```diff
@@ -142,6 +143,8 @@ impl StorageManager {
 
         storage.iter_store_from(metric_lifetime, &store_name, None, &mut snapshotter);
 
+        record_coverage(metric_id);
```

Comment:
The proposal said `snapshot_metric` can be called from one non-test place. Do we need to handle this here?